src/app/auth/login/login.component.html

```diff
@@ -1,24 +1,32 @@
 <div class="login-container">
   <app-window icon="🔑" label="Sign On">
     <div class="login-window-pane">
-
       <div class="under-ridged logo-container">
         <div class="logo-icon">🏃</div>
         <div class="logo-label">Angular Instant Messenger</div>
       </div>
       <form class="form" [formGroup]="loginForm" (ngSubmit)="login()">
-        <label for="screenName">
-          ScreenName🔑
-        </label>
-        <input required id="screenName" formControlName="username">
-        <label for="password">
-          Password
-        </label>
-        <input required id="password" type="password" formControlName="password">
-        <button class="grooved icon-button" type="submit" [disabled]="loginForm.invalid">
+        <label for="screenName"> ScreenName🔑 </label>
+        <input required id="screenName" formControlName="username" />
+        <label for="password"> Password </label>
+        <input
+          required
+          id="password"
+          type="password"
+          formControlName="password"
+        />
+        <button
+          class="grooved icon-button"
+          type="submit"
+          [disabled]="loginForm.invalid"
+        >
           <div class="icon-button__icon">🏃</div>
           <div class="icon-button__label">Sign On</div>
         </button>
+        <p class="text-muted text-center mt-3 mb-0">
```

Comment:
Consider following the 'Get Screenname'
https://jan.ucc.nau.edu/~lrm22/technology/macosx/video_chat/aim_classic/aim_setup.htm